resolwe_bio/tests/workflows/test_rna_seq.py

```diff
@@ -66,7 +66,30 @@ def test_bbduk_star_htseq_workflow(self):
             self.assertStatus(data, Data.STATUS_DONE)
 
         workflow = Data.objects.last()
-        self.assertFile(workflow, 'rc', 'workflow_reads_rc.tab.gz', compression='gzip')
+        self.assertFile(workflow, 'exp', 'workflow_reads_cpm.tab.gz', compression='gzip')
+
+    @tag_process('workflow-bbduk-star-htseq-paired')
+    def test_bbduk_star_htseq_paired_workflow(self):
+        with self.preparation_stage():
+            paired_reads = self.prepare_paired_reads(['SRR2124780_1 1k.fastq.gz'], ['SRR2124780_2 1k.fastq.gz'])
+            annotation = self.prepare_annotation('HS chr21_short.gtf.gz')
+            star_index_fasta = self.prepare_adapters('HS chr21_ensembl.fa.gz')
+            inputs = {'annotation': annotation.id, 'genome2': star_index_fasta.id}
+            star_index = self.run_process('alignment-star-index', inputs)
+            adapters = self.prepare_adapters()
+
+        inputs = {
+            'reads': paired_reads.id,
+            'adapters': [adapters.id],
+            'star_index': star_index.id,
+            'annotation': annotation.id,
+            'stranded': 'yes',
+        }
+        self.run_process('workflow-bbduk-star-htseq-paired', inputs)
+        for data in Data.objects.all():
+            self.assertStatus(data, Data.STATUS_DONE)
+        workflow = Data.objects.last()
+        self.assertFile(workflow, 'exp', 'workflow_ccshs_htseq_cpm.tab.gz', compression='gzip')
```

Comment:
Use file names that reflect what is being tested. `ccshs` comes from `custom-cutadapt`, `star`, `htseq-count`, `single` which is very different from what is asserted here.